src/transformers/models/llava/modeling_llava.py

```diff
@@ -240,6 +299,10 @@ def __init__(self, config: LlavaConfig):
         self.vision_tower = AutoModel.from_config(config.vision_config)
 
         self.multi_modal_projector = LlavaMultiModalProjector(config)
+
+        if config.use_image_newline_parameter:
+            self.image_newline = nn.Parameter(torch.empty(config.text_config.hidden_size, dtype=self.dtype))
```

Comment:
What was the conclusion about adding parameters in the other model PR discussion?

Reply:
This PR proposes to add support for llava 1.6 in modeling_llava.py. The only new trainable parameter is this one